ivy_tests/test_ivy/helpers/testing_helpers.py

```diff
@@ -376,6 +377,7 @@ def handle_frontend_test(
     test_inplace=BuiltInplaceStrategy,
     as_variable_flags=BuiltAsVariableStrategy,
     native_array_flags=BuiltNativeArrayStrategy,
+    gen_frontend_arrays=BuiltFrontendArrayStrategy,
```

Comment:
It's better if we use the full `generate`

Reply:
sure